projects/envoy/build.sh

```diff
@@ -71,9 +71,10 @@ do
 done
 
 # Build driverless libraries.
+# Benchmark about 1.5-2 GB per CPU
 bazel build --verbose_failures --dynamic_mode=off --spawn_strategy=standalone \
   --discard_analysis_cache --notrack_incremental_state --nokeep_state_after_build \
-  --local_cpu_resources=HOST_CPUS*0.75 \
+  --local_cpu_resources=16 \
```

Comment:
do you want to try something like `$(nproc) / 2`  to avoid having hardcoded value?